### PR TITLE
refactor: wallet text cleanup

### LINKED ISSUE #755: Final text review on Wallet
[FRO-132](https://linear.app/fuel-network/issue/FRO-132/final-text-review-on-wallet)

Review all texts on the Wallet to improve;

* Clarity
* Brevity
* Action-oriented language
* Maintain consistent terminology
* Consistent voice tone

### PR DESCRIPTION
**Pull request by sarahschwartz**
[FRO-132](https://linear.app/fuel-network/issue/FRO-132/final-text-review-on-wallet)
 close #755 
Some small changes to the text in the wallet to make it more concise and readable

### CODE REVIEW

**Review by luizstacio — CHANGES_REQUESTED** [review]
Just a small comment. Thanks for the review.

File: .changeset/shaggy-pots-shop.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (luizstacio) [inline .changeset/shaggy-pots-shop.md:2]: Can we add a patch update here to show the changes on text on the changelog?

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Review by tomiiide — APPROVED** [review]
Nice work @sarahschwartz .
Looks good.

File: packages/app/src/systems/Settings/components/ConnectionItem/ConnectionItem.tsx [hunk 2]
```diff
@@ -77,7 +77,9 @@ export const ConnectionItem: ConnectionItemComponent = ({
       <Avatar name={origin} src={favIconUrl} css={styles.avatar} />
       <Box css={styles.text}>
         <Text>{truncate(parseUrl(origin))}</Text>
-        <Text>{accounts} account(s) connected</Text>
+        <Text>
+          {accounts} {accounts > 1 ? 'accounts' : 'account'} connected
```
Comment (eswarasai) [inline packages/app/src/systems/Settings/components/ConnectionItem/ConnectionItem.tsx:81]:
```suggestion
          {accounts} {`account${accounts > 1 ? 's' : ''}`} connected
```

**Review by luizstacio — APPROVED** [review]

File: packages/app/src/systems/Account/pages/ExportAccount/ExportAccount.tsx [hunk 3]
```diff
@@ -34,7 +42,7 @@ export const ExportAccount = () => {
           <Stack gap="$4">
             <AccountItem account={account} />
             <Text css={styles.keyHeaderText}>
-              Using this Private Key, you can restore your account later. Make
+              With this Private Key, you can restore your account later. Make
```
[the remark this answers is not in the document]
Comment (sarahschwartz) [reply packages/app/src/systems/Account/pages/ExportAccount/ExportAccount.tsx]: What do you think about this? wallet screenshot

**Pull request merged**